Library/Formula/ucommon.rb

```diff
@@ -14,16 +12,29 @@ class Ucommon < Formula
   depends_on "pkg-config" => :build
   depends_on "gnutls"
 
+  needs :cxx11
+
+  # Upstream bug report:
+  #   http://lists.gnu.org/archive/html/bug-commoncpp/2015-01/msg00000.html
```

Comment:
It’s intentional, to show that this line is linked to the previous one :smiley: 

e.g.:

```
# something:
#   foo
#   bar
# something else:
#   qux
```

vs:
```
# something:
# foo
# bar
# something else:
# qux
```